.gitlab-ci.yml

```diff
@@ -38,6 +38,25 @@ default:
         - cd ${CORE_BUILD_DIR}
         - make -j $(nproc) check
 
+.test-pmix:
+    extends: .lc-variables
+    variables:
+        PYTHON: "/usr/bin/python3"
+        debug: t
+        FLUX_TESTS_LOGFILE: t
+    script:
+        - export FTC_DIRECTORY=$(pwd)
+        - !reference ['.build-core', 'script']
+        - export PKG_CONFIG_PATH=${CORE_INSTALL_PREFIX}/lib/pkgconfig:$(pkg-config --variable pc_path pkg-config)
+        - module load openmpi
+        - export PKG_CONFIG_PATH=$(dirname $(which mpicc))/../lib/pkgconfig:${PKG_CONFIG_PATH} 
+        - cd ${FTC_DIRECTORY}
+        - git clone https://github.com/flux-framework/flux-pmix
+        - cd flux-pmix
+        - ./autogen.sh
+        - PKG_CONFIG_PATH=${PKG_CONFIG_PATH} ./configure
```

Comment:
Does this work without `PKG_CONFIG_PATH=${PKG_CONFIG_PATH}`? Typically, assigning a variable to itself doesn't do anything, but if this doesn't work without it (especially after you've exported `PKG_CONFIG_PATH` above), then maybe there is something to look into here.

Reply:
In my script, PKG_CONFIG_PATH was not exported so I needed that to get it set in the  `./configure` environment.

Reply:
Ok, it is exported here which is why I was slightly confused, but isn't harmful so I'll leave it up to @wihobbs if he wants to do anything about it.

Reply:
Sorry - I didn't look beyond the quoted snippet :-) I agree, one or the other, not both.

Reply:
Dropped it.